tests/conftest.py

```diff
@@ -1,27 +1,39 @@
 import os
-import shutil
+import tempfile
 
 import pandas as pd
 import pytest
 
 import audformat.testing
 
 
-pytest.ROOT = os.path.dirname(os.path.realpath(__file__))
+@pytest.fixture(scope='package', autouse=True)
+def prepare_tests():
 
-pytest.DB = audformat.testing.create_db()
-pytest.DB_ROOT = os.path.join(pytest.ROOT, 'db')
-audformat.testing.create_attachment_files(pytest.DB, pytest.DB_ROOT)
-pytest.DB.save(pytest.DB_ROOT)
+    # Prepare files used in tests
+    # and set a temporary working directory
+    with tempfile.TemporaryDirectory() as tmp:
 
-pytest.FILE_DUR = pd.to_timedelta('1s')
+        current_dir = os.getcwd()
+        os.chdir(tmp)
+
+        pytest.DB.save(pytest.DB_ROOT)
+
+        audformat.testing.create_audio_files(
+            pytest.DB,
+            pytest.DB.root,
+            file_duration=pytest.FILE_DUR,
+        )
+        audformat.testing.create_attachment_files(
+            pytest.DB,
+            pytest.DB.root,
+        )
 
+        yield
 
-@pytest.fixture(scope='session', autouse=True)
-def create_audio_files():
-    audformat.testing.create_audio_files(
-        pytest.DB,
-        file_duration=pytest.FILE_DUR,
-    )
-    yield
-    shutil.rmtree(pytest.DB_ROOT)
+        os.chdir(current_dir)
+
+
+pytest.DB = audformat.testing.create_db()
+pytest.DB_ROOT = 'db'
+pytest.FILE_DUR = pd.to_timedelta('1s')
```

Comment:
I find it easier to read if the variables are defined before the fixtures.

Reply:
ok, swapped